loky/backend/semaphore_tracker.py

```diff
@@ -73,11 +72,13 @@ def ensure_running(self):
                 # process will out live us, so no need to wait on pid
                 exe = spawn.get_executable()
                 args = [exe] + util._args_from_interpreter_flags()
-                if len(args[-1]) > 100:  # pragma: no cover
-                    warnings.warn(
-                        "args_from_flag returned -R+ -> why??", RuntimeWarning)
```

Comment:
Ok apparently it also occured on python2.7 so it is probably not the cause of the `test_process` failures